src/gui/DatabaseOpenWidget.cpp

```diff
@@ -296,6 +303,14 @@ void DatabaseOpenWidget::yubikeyDetected(int slot, bool blocking)
     }
 }
 
+void DatabaseOpenWidget::yubikeyDetectComplete()
```

Comment:
@droidmonkey just a nitpick : why not `yubikeyDetectionCompleted`? Would be coherent with the style of the other event handlers.